src/Microsoft.ML.Data/DataView/DataViewConstructionUtils.cs

```diff
@@ -269,6 +269,10 @@ private Delegate CreateGetter(DataViewType colType, InternalSchemaDefinition.Col
                         return Utils.MarshalInvoke(delForKey, keyRawType, peek, colType);
                     }
                 }
+                else if (TypeManager.GetRawTypeOrNull(colType) != null)
+                {
+                    del = CreateDirectGetterDelegate<int>;
```

Comment:
>int [](start = 53, length = 3)

why int?

Reply:
It can be anything you want. The generic type will be over-write at the end of this function.

---
In reply to: [284945609](https://github.com/dotnet/machinelearning/pull/3263#discussion_r284945609) [](ancestors = 284945609)